includes/Dashboard.php

```diff
@@ -213,7 +213,7 @@ public function load_stories_dashboard() {
 
 		$preload_data = array_reduce(
 			$preload_paths,
-			'rest_preload_api_request',
+			__NAMESPACE__ . '\rest_preload_api_request',
```

Comment:
This means we can remove the `$_GET['_embed']` stuff here

Reply:
No, it still needed for this line 
https://github.com/google/web-stories-wp/blob/ad45bb9d602bea643d45dbbfa2e5378e3800551a/includes/REST_API/Stories_Controller.php#L370

Reply:
^ But that line can then be changed to determine `_embed` based on the `$request`, no?

Reply:
Yeah and I like that much better. I will make the change.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/5149/commits/368e672d8c658eac7ce82acbc18fb5baf01aa550